test/api/handlers/projects/publish.js

```diff
@@ -0,0 +1,68 @@
+var Lab = require('lab');
+var lab = exports.lab = Lab.script();
+
+var experiment = lab.experiment;
+var test = lab.test;
+var before = lab.before;
+var after = lab.after;
+var expect = require('code').expect;
+
+var config = require("../../../lib/fixtures/files").update;
```

Comment:
This is supposed to contain the fail or success options being passed to the server in each test. This is pulling from the `files` fixtures, so these tests will behave strangely.